src/scripts/main.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, (e) => {
+      resolve(`It was ${eventName}
```

Comment:
There is a syntax error in the template literal. The string is not properly closed. Ensure that the backtick (`) at the end of the string is added to close the template literal correctly.